composer.json

```diff
@@ -35,7 +35,7 @@
         "cache/integration-tests": "^0.16",
         "predis/predis":           "^1.0",
         "symfony/cache":           "dev-master",
-        "illuminate/cache":        "^5.0"
+        "illuminate/cache":        "^5.4"
```

Comment:
This version bump is needed so we can receive cache flush feedback https://github.com/illuminate/cache/commit/8c71f266e9206354c8957bac4a4e571fa64ed4eb